components/create-post-page/UploadTextbookCover.tsx

```diff
@@ -0,0 +1,44 @@
+import { Box, Center, Icon, Input, Text, VStack } from "@chakra-ui/react";
+import { UploadIcon } from "@heroicons/react/outline";
+import styles from "@styles/upload.module.css";
+import { ChangeEvent } from "react";
+
+interface Props {
+  onCoverPhotoUploaded: (coverPhoto: Blob) => void;
+};
+
+export default function UploadTextbookCover({
```

Comment:
We should probably use the `ImageUploadModal` component here so we can standardize on image aspect ratio/resizing

Reply:
Okay. I think I will have to refactor some stuff. It doesn't make sense to have the image upload as a modal in this context. I'll look into extracting the upload and resizing tool into a non-modal component.

Reply:
I think it's fine in a modal but feel free to play with it